### PR TITLE
feat

### PR DESCRIPTION
**Pull request by ckahub**
created wishlist page
chrome-capture-2022-2-26 (5)

### CODE REVIEW

**Review by ravikumar0403 — APPROVED** [review]
Looks good to merge.. Just make sure you make the changes mentioned in last PR

File: style/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    height: 1rem;
+    line-height: 1rem;
+    justify-content:space-evenly;
+    align-items:center;
+    box-shadow: none;
+    text-align: center;
+}
+.nav2 ul{
+    list-style: none;
+    /* text-align: center; */
```
Comment (dusariudaykumar) [inline style/wishlist.css:33]: remove comments

File: style/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    color: var(--secondary-color);
+    font-size: medium;
+    display:flex;
+    align-items:center;
+    justify-content: center;
+    text-align: center;
+}
+
+footer{
+    background-color: #172b4df3;
```
Comment (JaswanthReddyPadala) [inline style/wishlist.css:136]: Use variable here for better code practice

File: style/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    background-color: tan;
+    color: var(--primary-color);
+    font-weight: bold;
+    font-family: Arial, Helvetica, sans-serif;
+    width: 14rem;
+    height: 2rem;
+}
+.add2cart:hover{
+    font-size: large;
+    color: #055252;
```
Comment (JaswanthReddyPadala) [inline style/wishlist.css:113]: Same here

**Review by dusariudaykumar — APPROVED** [review]
The naming conventions are good.

Looks good to merge

**Review by JaswanthReddyPadala — APPROVED** [review]
Rest of them looks good, UI can also be improved
GTG ✅🚀

**Pull request merged**